changelog.txt

```diff
@@ -2,6 +2,7 @@
 
 = 1.25.7 - 2021-xx-xx =
 * Fix	- Prevent error notices on checkout page load
+* Tweak	- Highlight rate call usage over limit on WooCommerce Shipping settings page.
```

Comment:
Nitpicking on formatting: Align the messages on Fix and Tweak. And not from this PR but missing a period on Fix.